scss/product/components/_dropdown.scss

```diff
@@ -9,13 +9,19 @@
   @include caret;
 }
 
+//three dot menu toggle for drop down
+.dot-menu{
+margin-top:-10%;
+cursor: pointer;
+}
+
 // The dropdown menu
 .dropdown-menu {
   position: absolute;
   top: 100%;
   left: 0;
   z-index: $zindex-dropdown;
-  display: none; // none by default, but block on "open" of the menu
+  // display: none; // none by default, but block on "open" of the menu
```

Comment:
display will be none for menu, by default.

to show the preview in <Playground> we can overwrite display with utils class 

eg
`<div class="dropdown-menu d-block">
`

Reply:
If we add util class to display in preview, It would also be visible in the code block.

Reply:
Added a separate class '`decode`' for displaying outside `playground` in style.scss